modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd

```diff
@@ -439,54 +402,42 @@ clean_names(test)
 This time lets also make it a smaller subset so it is easier for us to see the full dataset as we work through examples. 
 
 ```{r}
-#install.packages(OCSdata)
-#library(OCSdata)
-#OCSdata::load_imported_data("ocs-bp-opioid-rural-urban")
+#read_csv("https://daseh.org/data/Colorado_ER_heat_visits_by_county_gender.csv")
```

Comment:
library(dasehr)
CO_heat_ER_bygender

?

Reply:
I will show both  if you want me to show the package- I was concerned we would have people who can't install packages from GitHub like we have had in the past

Reply:
I think we should have a preference for the package, but leaving a note that you can also do this makes sense!